packages/zowe-explorer-api/README.md

```diff
@@ -5,11 +5,9 @@
 
 Extensibility API for Zowe Explorer is a collection of APIs that can be used to extend the [Zowe Explorer](https://github.com/zowe/zowe-explorer-vscode) VS Code extension with alternative z/OS interaction protocols and new capabilities.
 
-The current state of this API is experimental, but the goal is provide a stabilized version that can be used for Zowe Conformance certifications in the future. See this issue for more details: <https://github.com/zowe/zowe-explorer-vscode/issues/837>.
+Extensibility API for Zowe Explorer is used by open source Zowe Explorer extensions, including the [IBM z/OS FTP for Zowe Explorer](../zowe-explorer-ftp-extension) and [IBM CICS for Zowe Explorer](https://github.com/zowe/cics-for-zowe-client/tree/main/packages/vsce) VS Code extensions, as well as a several commercial extensions.
```

Comment:
good catch on this, I agree that we are in a stable state now with v2 & v3 😄 

grammar edit needed at the end of sentence `..., as well as a several commercial extension.` to `..., as well as several commercial extension.`

Reply:
good catch 👍